documentation.md

```diff
@@ -0,0 +1,233 @@
+| patchable                 | Bool          | Boolean checking whether if the update is patchable via Cyberwatch or not| true       |
+| cve_announcements         | Object        | List of the CVEs concerned by the update| [CveAnnouncements](#Cve-announcements)      |
+| current                   | Object        | Vulnerable version of the package affected by the update| [Current](#Current)         |
+| target                    | Object        | Version of the package to update to| [Target](#Current)                               |
+| created_at                | String (date) | Creation date of the update in Cyberwatch| "2019-04-09T17:13:07.000+02:00"            |
+| updated_at                | String (date) | Last modification date of the update in Cyberwatch| "2019-04-09T17:13:07.000+02:00"   |
+
+### Cve
+
+| Attribute                 | Type          | Description                       | Example of possible value                         |
+|---------------------------|:-------------:|:---------------------------------:|---------------------------------------------------|
+| cve_code                  | String        | CVE code                          | "CVE-2019-5953"                                   |
+| content                   | String        | CVE description                   | "In systemd before v242-r…her than \"allow_any\"."|
+| published                 | String        | Publication date of the CVE by the authorities| "2019-04-09T23:29:03.000+02:00"       |
+| last_modified             | String        | Last modification date of the CVE by the authorities| "2019-04-26T16:45:22.000+02:00" |
+| cve_score                 | Float         | CVE score                         | 4.4                                               |
+| created_at                | String (date) | Creation date of the CVE in Cyberwatch| "2019-04-08T22:17:34.000+02:00"               |
+| updated_at                | String (date) | Last modification date of the CVE in Cyberwatch| "2019-04-29T09:33:37.000+02:00"      |
+
+#### Current
+
+| Attribute                 | Type          | Description                       | Example of possible value                         |
+|---------------------------|:-------------:|:---------------------------------:|---------------------------------------------------|
+| vendor                    | String        | Package vendor                    | null                                              |
+| product                   | String        | Package name                      | "libsystemd0"                                     |
+| hash_index                | String (hash) | Hash identifier                   | "13a61dd2bfe0fbb063a97b66c6234759"                |
+| type                      | String        | Package type in Cyberwatch        | "Packages::Deb"                                   |
+| version                   | String        | Package version                   | "237-3ubuntu10.13"                                |
```

Comment:
Missing documentation of the `Node` object : https://github.com/Cyberwatch/cyberwatch_api_toolbox/blob/e532605342d0c1b0a19f1ede63559952c3a25ad5/cbw_api_toolbox/cbw_objects/cbw_node.py